src/js/common/schemaform/FormSaved.jsx

```diff
@@ -41,10 +41,15 @@ class FormSaved extends React.Component {
       <div>
         <div className="usa-alert usa-alert-info">
           <div className="usa-alert-body">
-            <strong>Your application has been saved!</strong><br/>
+            <strong>Your application has been saved.</strong><br/>
             {!!lastSavedDate && <p>Last saved on {moment(lastSavedDate).format('M/D/YYYY [at] h:mma')}.</p>}
             {success}
-            If you're on a public computer, please sign out before you leave to ensure your data is secure.
+            If you're logged in through a public computer, please sign out of your account before you log off to keep your information secure.
+          </div>
+        </div>
+        <div className="usa-alert usa-alert-warning">
+          <div className="usa-alert-body">
+            <strong>Please verify your identity.</strong> We want to provide you with the highest level of security to keep your information safe. Please go to your account (link here) and verify your identity.
```

Comment:
Also, it looks like there are two options for the link here, either:
-We can have a link that starts the verification process directly, we don't have to send them to the profile page. Either way works, just wanted to point that out in case that was preferred. (As proposed by @jbalboni [here](https://github.com/department-of-veterans-affairs/vets.gov-team/issues/4196#issuecomment-320063431))
- or as already phrased above
Which option seems preferable? And what would be the actual link to provide?